src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -15,7 +15,15 @@ object CromwellServer extends WorkflowManagerSystem {
   val conf = ConfigFactory.load()
   val service = actorSystem.actorOf(CromwellApiServiceActor.props(workflowManagerActor, conf), "cromwell-service")
   val webserviceConf = conf.getConfig("webservice")
-  service.bindOrShutdown(interface = webserviceConf.getString("interface"), port = webserviceConf.getInt("port")) onSuccess {
-    case _ => actorSystem.log.info("Cromwell service started...")
+
+  def run(): Int = {
+    val interface = webserviceConf.getString("interface")
+    val port = webserviceConf.getInt("port")
+    val futureBind = service.bindOrShutdown(interface = interface, port = port)
```

Comment:
Not new with these changes, but over in Lenthallshire I see this in `bindOrShutdown`:

```scala
actorSystem.log.error(throwable, s"Binding failed interface $interface port $port")
actorSystem.shutdown()
```

Given what we now know about the asynchronicity of Akka logging, it's not too hard to imagine that this error message will never be emitted.